client/src/components/list/BookCardLists.jsx

```
@@ -1,7 +1,13 @@
+import React from 'react';
+import styles from './BookCardLists.module.css';
+import BookCard from '../card/BookCard';
 
-export default function BookCardLists() {
+export default function BookCardLists({ bookCardLists }) {
   return (
-    <div>BookCardLists</div>
-  )
+    <div className={styles.wrapper}>
+      {bookCardLists?.map((bookDetail) => {
+        return <BookCard bookDetail={bookDetail}></BookCard>;
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)